scripts/init_letsencrypt.sh

```diff
@@ -7,12 +7,12 @@ set -o allexport
 source .env
 set +o allexport
 
-CONFIG_PATH="${CONFIG_PATH:-'./conf'}"
+SCRIPT_DIR=$(dirname "$(realpath "$0")")
```

Comment:
sorry forgot one thing:

Here we should use already something like

`ROOT_DIR="../${SCRIPT_DIR}"` but in one expression if possible

instead od having to `..` all the time after